File: ProblemSet/1503. 找零/1503. 找零.java
public class Solution {
    /**
     * @param amount: The amount you should pay.
     * @return: Return the minimum number of coins for change.
     */
    public int giveChange(int amount) {
        int[] coins = {64, 16, 4, 1};
        int change = 1024 - amount;
        int num = 0;
        for (int coin : coins) {
            num += change / coin;
            change %= coin;
        }
        return num
    }
}

File: ProblemSet/1503. 找零/1503. 找零.py
class Solution:
    """
    @param amount: The amount you should pay.
    @return: Return the minimum number of coins for change.
    """
    def giveChange(self, amount):
        coins = [64, 16, 4, 1]
        change = 1024 - amount
        num = 0
        for coin in coins:
            num += change // coin
            change %= coin
        return num
